server/src/main/java/org/elasticsearch/transport/SniffConnectionStrategy.java

```diff
@@ -64,6 +64,13 @@ public class SniffConnectionStrategy extends RemoteConnectionStrategy {
         }, new StrategyValidator<>(ns, key, ConnectionStrategy.SNIFF), Setting.Property.Dynamic, Setting.Property.NodeScope)
     );
 
+    public static final Setting.AffixSetting<String> REMOTE_CLUSTER_AUTHORIZATION = Setting.affixKeySetting(
+        "cluster.remote.",
+        "authorization",
```

Comment:
Can we update the PR description to match the name here (is upper case in PR description)?

Reply:
The description says the name is case-insensitive, but I agree updating the PR description could make it clearer.

Reply:
Done.